tox.ini

```diff
@@ -57,6 +57,7 @@ commands =
 [testenv:pypy]
 basepython = pypy
 commands =
+    pip install PyPyDispatcher>=2.0.6
```

Comment:
I tried installing universal wheel on ancient ``pip 1.5.4``, ``setuptools 3.3`` (default in Ubuntu 14.04), and also on a couple of other random combinations (setuptools 17.0, 18.0, pip 7.1.2, 8.1.1), and all seems to work fine. Another thing we could do for extra safety is include a check for the setuptools version like this: https://github.com/PyCQA/astroid/pull/360/files

To summarize, it looks like it's fine to use the version marker, but it's really hard to check all possible combinations.